docs/FAQ.md

```diff
@@ -1,116 +1,57 @@
-# Frequently asked questions about AFL++
-
-## Contents
-
-  * [What is the difference between AFL and AFL++?](#what-is-the-difference-between-afl-and-afl)
-  * [I got a weird compile error from clang](#i-got-a-weird-compile-error-from-clang)
-  * [How to improve the fuzzing speed?](#how-to-improve-the-fuzzing-speed)
-  * [How do I fuzz a network service?](#how-do-i-fuzz-a-network-service)
-  * [How do I fuzz a GUI program?](#how-do-i-fuzz-a-gui-program)
-  * [What is an edge?](#what-is-an-edge)
-  * [Why is my stability below 100%?](#why-is-my-stability-below-100)
-  * [How can I improve the stability value?](#how-can-i-improve-the-stability-value)
+# Frequently asked questions (FAQ)
 
 If you find an interesting or important question missing, submit it via
-[https://github.com/AFLplusplus/AFLplusplus/issues](https://github.com/AFLplusplus/AFLplusplus/issues)
+[https://github.com/AFLplusplus/AFLplusplus/discussions](https://github.com/AFLplusplus/AFLplusplus/discussions).
 
-## What is the difference between AFL and AFL++?
+### General
```

Comment:
general, targets and performance headers have 3 # .... shouldnt it be 2 # ?

Reply:
To fix that and to avoid the redundant questions, I went with another approach:
https://github.com/llzmb/AFLplusplus/blob/docs_edit_faq/docs/FAQ.md

What do you think?

Reply:
I think it is very readable in MD, and still readable with a text editor so that is a good solution. what is not possible anymore though is to have a link to a specific entry from a different document, or is that still somehow possible?

Reply:
Good point! I can add an ID to each summary tag. It makes the MD a little less human-readable, but it should work for internal and external linking:
```html
<details>
  <summary id="where-can-i-find-tutorials">Where can I find tutorials?</summary><p>

  We compiled a list of tutorials and exercises, see [tutorials.md](tutorials.md).
</p></details>
```

```markdown
[internal link](#where-can-i-find-tutorials)
[external link](FAQ.md#where-can-i-find-tutorials)
```

What do you think about this solution?

Reply:
I think this is good. @domenukk / @andreafioraldi  ?